modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2DeviceFlowTestCase.java

```diff
@@ -317,19 +328,34 @@ public HttpResponse sendGetRequest(HttpClient client, String locationURL) throws
     /**
      * Create Application with the given app configurations
      *
-     * @return OAuthConsumerAppDTO
-     * @throws Exception
+     * @return ApplicationResponseModel
+     * @throws Exception exception
```

Comment:
Shall we fix the formatting?

Reply:
Fixed with https://github.com/wso2/product-is/pull/20096